browser/android/rewards/safetynet_check.h

```diff
@@ -0,0 +1,53 @@
+/* Copyright (c) 2019 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_BROWSER_ANDROID_REWARDS_SAFETYNET_CHECK_H_
+#define BRAVE_BROWSER_ANDROID_REWARDS_SAFETYNET_CHECK_H_
+
+#include <jni.h>
+#include <memory>
+#include <string>
+#include <vector>
+
+#include "base/android/scoped_java_ref.h"
+#include "net/base/completion_once_callback.h"
+
+namespace safetynet_check {
+
+class SafetyNetCheckRunner;
+using ClientAttestationCallback =
+  base::OnceCallback<void(bool, const std::string&)>;
+
+class SafetyNetCheck {
+ public:
+    explicit SafetyNetCheck(SafetyNetCheckRunner* runner);
```

Comment:
Just curioius, if it is possible to keep C++ files conformant to CC? i.e. do `clang-format` on everything, add `DISALLOW_COPY_AND_ASSIGN`, etc